sierra_adapter/terraform/sierra_merger/services.tf

```diff
@@ -0,0 +1,52 @@
+module "appautoscaling" {
+  source = "git::https://github.com/wellcometrust/terraform.git//autoscaling/app/ecs?ref=v1.1.0"
+  name   = "sierra_${var.resource_type}_merger"
+
+  cluster_name = "${var.cluster_name}"
+  service_name = "${module.sierra_merger_service.service_name}"
+}
+
+module "sqs_autoscaling_alarms" {
+  source = "git::https://github.com/wellcometrust/terraform.git//autoscaling/alarms/sqs?ref=v1.1.0"
+  name   = "sierra_${var.resource_type}_merger"
+
+  queue_name = "${module.update_events_queue.name}"
+
+  scale_up_arn   = "${module.appautoscaling.scale_up_arn}"
+  scale_down_arn = "${module.appautoscaling.scale_down_arn}"
+}
+
+module "sierra_merger_service" {
+  source             = "git::https://github.com/wellcometrust/terraform.git//services?ref=s3-mystery"
+  name               = "sierra_${var.resource_type}_merger"
+  cluster_id         = "${var.cluster_id}"
+  task_role_arn      = "${module.ecs_sierra_merger.task_role_arn}"
+  vpc_id             = "${var.vpc_id}"
+  app_uri            = "${var.ecr_repository_url}:${var.release_id}"
+  nginx_uri          = "${var.nginx_ecr_repository_url}:${var.nginx_release_id}"
```

Comment:
Don't think we need the `nginx_uri` as this is defaulted if we use `v1.3.0` of the services module.

Reply:
yep, removing now